functions.php

```diff
@@ -129,6 +135,20 @@ function twentynineteen_scripts() {
 }
 add_action( 'wp_enqueue_scripts', 'twentynineteen_scripts' );
 
+/**
+ * Fix skip link focus in IE11.
+ *
+ * This does not enqueue the script because it is tiny and because it is only for IE11,
+ * thus it does not warrant having an entire dedicated blocking script being loaded.
+ *
+ */
+function _twentynineteen_skip_link_focus_fix() {
```

Comment:
This change shouldn't be included _yet_. It's dependent on #47. Also, once/if that PR is merged, this function will then need the `is_amp` as seen in the WP Rig PR: https://github.com/wprig/wprig/pull/139

Reply:
@westonruter should we just revert back to the previous for now until #47 is merged?

Reply:
yes